_database-integrations/amazon-s3/amazon-s3-csv-latest.md

```diff
@@ -0,0 +1,630 @@
+              <td>
+                <strong>type</strong>
+              </td>
+              <td>
+                <strong>magic</strong>
+              </td>
+            </tr>
+            {% for record in second-job %}
+            <tr>
+              <td>
+                {{ record.primary-key }}
+              </td>
+              <td>
+                {{ record.name }}
+              </td>
+              <td>
+                {{ record.type }}
+              </td>
+              <td>
+                {{ record.magic }}
+              </td>
+            </tr>
+            {% endfor %}
+          </table>
+
+      - title: "Querying Append-Only tables"
+        anchor: "querying-append-only-tables"
+        content: |
+          Querying Append-Only tables requires a different strategy than you might normally use. For instructions and a sample query, check out the [Querying Append-Only tables guide]({{ link.replication.append-only-querying | prepend: site.baseurl }}).
+
+          {% capture append-only-destinations %}
+          The answer depends on your destination:
+
+          - **BigQuery and Amazon S3 CSV** - These destinations are Append-Only regardless of whether Primary Keys are specified.
+
+             If you specified Primary Keys during setup, use the column(s) specified (ex: `id`). Otherwise, use `{{ system-column.primary-key }}`.
+          - **All other destinations** - Use `{{ system-column.primary-key }}` when querying Append-Only tables.
+          {% endcapture %}
+
+          {% include note.html first-line="**What column do I use as a Primary Key when querying?**" content=append-only-destinations %}
+          
+  - title: "Determining data types"
+    anchor: "determining-data-types"
+    content: |
+      To determine a column's data type, Stitch will analyze the first 1,000 lines of (up to) the first **five** files included for a given table.
+
+      Stitch's {{ integration.display_name }} integration will load data from CSV files and type it as one of the following data types:
+
+      {% for item in integration.loading-data-types %}
+      - `{{ item.name | upcase }}`{% if item.note %} - {{ item.note }}{% endif%}
+      {% endfor %}
+
+
+# -------------------------- #
+#         Schema Info        #
+# -------------------------- #
+
+schema-sections:
+  - content: |
+      In this section:
+
+      - [How multiple files are mapped to a table](#mapping-files-to-table)
+      - [Why column names may be transformed](#column-name-transformations)
+
+  - title: "Mapping files to a single table"
+    anchor: "mapping-files-to-table"
+    content: |
+      [CONTENT]
+
+  - title: "Column name transformations"
+    anchor: "column-name-transformations"
+    content: |
+      When loading data, Stitch may perform some light transformation on column names to ensure the names follow the destination's rules for column names. This might include removing or replacing spaces or illegal characters such as `!#*`.
+
```

Comment:
I know we just told you that this was going to be something we wanted to mention in the docs, but I'm pretty sure we decided today around 10:45 that we were going to remove the column name changes and just leave them as they are in the file.

Reply:
@HarrisonMarcRose Yeah - but what about how the loaders themselves work?